docs/topics/upgrading.md

```diff
@@ -97,41 +97,6 @@ In some newer images, the default value of upload images is written inside the c
 
 Before we do the actual upgrade, we need to stop the containers and remove the volume that is shared between the `wikibase` and `wikibase_jobrunner` containers.
 
-### Review the mounts currently used by the wikibase container
```

Comment:
I feel like I'm wildly behind the official update process but what makes this no longer necessary? Failing to remove this volume would presumably prevent "recreation" of the LocalSettings?

Reply:
> Failing to remove this volume would presumably prevent "recreation" of the LocalSettings

Why do we want LocalSettings.php to be recreated?
We do not want install.php to run a second time, so LocalSettings.php needs to remain?